tests/test_units.py

```diff
@@ -69,7 +77,7 @@ def test_get_unit(self):
 
         # Get CellML unit
         store = UnitStore()
-        assert str(store.get_unit('liter')) == 'liter'
+        assert str(store.get_unit('liter')).endswith('liter')
```

Comment:
Except I thought CellML units didn't get prefixes, so this should still equal 'liter'?

Reply:
Good point!